scripts/ci/compose.yml

```diff
@@ -0,0 +1,8 @@
+name: ceno
+services:
+  runner:
+    restart: always
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file: .env
```

Comment:
We should probably put the `.env` in the `.gitignore`, so people don't commit one by accident?

Reply:
yeah you are right